### PR TITLE
Corrects spelling in comments

### PR DESCRIPTION
**Pull request by BaronGreenback**
Got bored - so corrected spelling mistakes throughout the code.

### CODE REVIEW

**Comment by BaronGreenback** [comment]
@crobibero In the process came across the following, which I assume is a bug??

LiveTVContollers: [HttpGet("Tuners/Discover")] (Typo on discovery).

**Comment by crobibero** [comment]
> 
> 
> @crobibero In the process came across the following, which I assume is a bug??
> 
> LiveTVContollers: [HttpGet("Tuners/Discover")] (Typo on discovery).

https://github.com/jellyfin/jellyfin/blob/907695dec7fda152d0e17c1197637bc0e17c9928/MediaBrowser.Api/LiveTv/LiveTvService.cs#L699

Sadly it's not something that happened during migration...

**Comment by BaronGreenback** [comment]
> > @crobibero In the process came across the following, which I assume is a bug??
> > LiveTVContollers: [HttpGet("Tuners/Discover")] (Typo on discovery).
> 
> https://github.com/jellyfin/jellyfin/blob/907695dec7fda152d0e17c1197637bc0e17c9928/MediaBrowser.Api/LiveTv/LiveTvService.cs#L699
> 
> Sadly it's not something that happened during migration...

I've corrected it in this PR - is that going to cause an issue?

File: Jellyfin.Api/Controllers/LiveTvController.cs [hunk 1]
```diff
@@ -1155,7 +1155,7 @@ public ActionResult<IEnumerable<NameIdPair>> GetTunerHostTypes()
         /// <param name="newDevicesOnly">Only discover new tuners.</param>
         /// <response code="200">Tuners returned.</response>
         /// <returns>An <see cref="OkResult"/> containing the tuners.</returns>
-        [HttpGet("Tuners/Discvover")]
+        [HttpGet("Tuners/Discover")]
```
Comment (crobibero) [inline Jellyfin.Api/Controllers/LiveTvController.cs:1158]:
```suggestion
        [HttpGet("Tuners/Discvover", Name = "DiscvoverTuners")]
        [HttpGet("Tuners/Discover")]
```

**Review by crobibero — APPROVED** [review]

**Review by joshuaboniface — APPROVED** [review]

**Pull request merged**